BookingsApi/BookingsApi.UnitTests/DAL/Queries/GetVhoNonAvailableWorkHoursQueryTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BookingsApi.DAL;
+using BookingsApi.DAL.Queries;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.DAL.Queries
+{
+    public class GetVhoNonAvailableWorkHoursQueryTest
```

Comment:
what are these tests for? We have integration tests at https://github.com/hmcts/vh-bookings-api/blob/master/BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetVhoNonAvailabilityWorkHoursQueryHandlerDatabaseTests.cs, which I think should be enough

Reply:
it is for the coverage

Reply:
deleted